api/serviceregistry/serviceregistry.proto

```diff
@@ -45,6 +45,7 @@ message WorkloadIdentifier {
   string cluster = 1;
   string podName = 2;
   string name = 3;
+  string nseName = 4;
```

Comment:
Why do we need both the `name` and 'nseName`?

Reply:
The idea is to track which NSE that a workload is connecting through so we can use the data to construct topology views.  The name is the "service" name that the workload is registering with and we use it for DNS reg.